src/Main.hs

```diff
@@ -19,27 +21,33 @@ import System.Console.GetOpt
 import System.Directory(getDirectoryContents)
 import Numeric(showHex)
 
-import Paths_graphmod (version)
+-- import Paths_graphmod (version)
 import Data.Version (showVersion)
 
 main :: IO ()
 main = do xs <- getArgs
           let (fs, ms, errs) = getOpt Permute options xs
+              opts = foldr ($) default_opts fs
           case errs of
             [] | show_version opts ->
-                  putStrLn ("graphmod " ++ showVersion version)
+                  putStrLn ("graphmod ")
 
-               | otherwise ->
-                  do (incs,inps) <- fromCabal (use_cabal opts)
-                     g <- graph (foldr add_inc (add_current opts) incs)
-                                (inps ++ map to_input ms)
-                     putStr (make_dot opts g)
-              where opts = foldr ($) default_opts fs
+               | otherwise -> do
```

Comment:
Could you avoid the inverted indentation so that code has a similar style to the rest of the program.